### ISSUE TITLE
YSlow not working on Firefox 37 with Firebug 2.0.7

### ISSUE THREAD

**Issue by ratan527** [marcelduran/yslow#153]
I have installed the Firefox 37.0.1 and Firebug 2.0.7 with the add-on YSlow 3.1.8.

I am using Windows 7 operating system.

I could not able to click on any of the links in YSlow. Could you please help me navigate through all the feaures of the yslow.

Thanks
Ratan

**Comment by 8la** [comment]
Same issue here

Windows8
Firefox 37.0.1
Yslow 3.1.8
Firebug 2.0.9

**Comment by 8la** [comment]
To discard conflicts with other plugins/extensions:
1.- Created new profile
2.- Installed Firebug and Yslow
3.- Restart Firefox 

Same result, cant get any reaction on the yslow interface. No button works...

**Comment by stevem** [comment]
+1

Xubuntu (3.13.0-49-generic #83-Ubuntu SMP Fri Apr 10 20:11:33 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux)
FF 37.0.1
Yslow 3.1.8
Firebug 2.0.9

Console shows "Error: Exposing privileged or cross-origin callable is prohibited" errs

**Comment by stevem** [comment]
So this might be a dupe of #133 ?

**Comment by PrashantChauhan** [comment]
Same issue here

Windows 2008 R2
Firefox 37/ 36
Yslow 3.1.8
Firebug 2.0.9

working in FIREFOX 35

**Comment by hendrickbert** [comment]
win 8.1 X64
FF 37.02
FB 2.0.9
Yslow 3.1.8

**Comment by THX2112** [comment]
Same issue.

Win 7
FF 37.0.2
FB 2.0.9
Yslow 3.1.8

None of the Yslow buttons or tabs work.

**Comment by Mark-Creeten** [comment]
Same here:

Mac Yosemite
Firefox 37.02
Firebug 2.0.9.1 -signed
Yslow 3.1.8

None of the Yslow buttons or tabs work.

**Comment by Taliman** [comment]
Not working either with FF 38.0  W 7  FB 2.0.9

**Comment by j-hernandez** [comment]
Confirmed also

OSX Yosemite
Firefox 37.0.2
Firebug 2.0.9
Yslow 3.1.8

Only UI element to respond in any way is the Rulesets dropdown, but net result is nil. All other elements unresponsive

**Comment by davidpricedev** [comment]
Same here: 
Win 7 
FF: 37.0.2
YS: 3.1.8
FB: 2.0.9

**Comment by themhz** [comment]
Same here., FF 38.0.1.  I am only using it in chrome were it works.

**Comment by loorlab** [comment]
Same here:

WIN7
Firefox 38.0.1
Firebug 2.0.9
Yslow 3.1.8

None of the Yslow buttons or tabs work.

**Comment by rogervila** [comment]
OS X 10.9.5 (Mavericks)
Firefox 38.0.5
Firebug 2.0.10
Yslow 3.1.8.1 - signed

Yslow tabs and buttons are not woking.

**Comment by cheshrkat** [comment]
Same here - OSX Yosemite, FF 38.0.5, Firebug 2.0.10, Yslow 3.1.8.1-signed.

**Comment by ghost** [comment]
Xubuntu 14.04
Firefox 38.0
Firebug 2.0.11
Yslow 3.1.8.1 - signed

Yslow tabs and buttons are not woking.

Is there any work being done to resolve this, or are we all just barking in the forest?

**Comment by aiac** [comment]
Yep, I cant use YSlow also. Same behaviour - tabs in YSlow not working, and links.

Win 8.1
FF 38.0.5
YSlow 3.1.8.1

**Comment by valnub** [comment]
Same issue here with FF38 and Win7.
Does this ticket get any attention? LIke this, the add-on is barely usable.

**Comment by Nickyvanrijsbergen** [comment]
Same issue. Is there a solution already?

**Comment by mauriciosoares** [comment]
Same here...

Yosemite 10.10.3
FF 39.0
YSlow 3.1.8.1

😰

Unfortunately the last commit in this repository was like 1 year ago... I'm not quite sure this bug will be solved any sooner...

**Comment by iamshreeram** [comment]
I'm seeing the same issue. Any solution available?
